frontend/src/components/pod/Details.tsx

```diff
@@ -500,11 +500,11 @@ export default function PodDetails(props: PodDetailsProps) {
           },
           {
             name: t('Host IP'),
-            value: item.status.hostIP,
+            value: item.status.hostIP || '',
```

Comment:
"new" recommended way of assigning defaults is to use `??` which handles edge cases better than `||`.
in this case it doesn't matter but it's a good habit 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing

Reply:
I know that but I tend to always use ||. I will change it though;